web/themes/contrib/civictheme/src/CivicthemeStylesheetParser.php

```diff
@@ -96,9 +96,7 @@ protected static function parseVariables($content): array {
     preg_match_all('/(--[a-zA-Z0-9-]+)\s*:\s*([^;]+);/i', $content, $matches, PREG_SET_ORDER);
 
     array_walk($matches, static function (array $value) use (&$variables): void {
-      if (!empty($value[1])) {
-        $variables[trim($value[1])] = empty($value[2]) ? NULL : trim($value[2]);
-      }
+      $variables[trim($value[1])] = empty($value[2]) ? NULL : trim($value[2]);
```

Comment:
And this change?

Reply:
https://github.com/civictheme/monorepo-drupal/pull/1294/commits/79f29d80e187bc11f10a61ea845ab7b901dd6d32

Reply:
Thanks